### PR TITLE
feat(ui): Add create org to side nav

### LINKED ISSUE #13158: Add create new org to nav
as a user,
when hovering over the settings wrench in the side nav,
i see the `<current org name> Settings` text
with an option beneath to Create Organization
which when clicked opens, the Create Organization overlay

### PR DESCRIPTION
**Pull request by ischolten**
Closes #13158

_Briefly describe your proposed changes:_
new_org

  - [ ] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergeable
  - [ ] Tests pass
  - [x] http/swagger.yml updated (if modified Go structs or API)
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/cypress/e2e/buckets.test.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ describe('Buckets', () => {
       const newBucket = '🅱️ucket'
       cy.getByTestID('table-row').should('have.length', 1)
 
-      cy.contains('Create').click()
+      cy.contains('Create Bucket').click()
```
Comment (alexpaxton) [inline ui/cypress/e2e/buckets.test.ts]: no `getByTestID` ?

File: ui/src/pageLayout/containers/Nav.tsx [hunk 2]
```diff
@@ -87,36 +91,38 @@ class SideNav extends PureComponent<Props, State> {
           active={getNavItemActivation(['tasks'], location.pathname)}
         />
         <NavMenu.Item
-          title="Settings"
+          title={`${_.startCase(orgName)} Settings`}
```
Comment (alexpaxton) [inline ui/src/pageLayout/containers/Nav.tsx]: I think `_.startCase` may sometimes alter the org name in undesirable ways

Comment (ischolten) [reply ui/src/pageLayout/containers/Nav.tsx]: I think it just ensures that the first letter of each word is capitalized

File: ui/cypress/e2e/buckets.test.ts (hunk 1 above)
Comment (ischolten) [reply ui/cypress/e2e/buckets.test.ts]: this test was just modified bc there were now two create buttons on the page, but i can change it to use the get by test id

File: ui/src/pageLayout/containers/Nav.tsx (hunk 2 above)
Comment (ischolten) [reply ui/src/pageLayout/containers/Nav.tsx]: removing for this pr

**Pull request merged**